proto/ibc/tendermint/tendermint.proto

```diff
@@ -68,6 +68,26 @@ message ConsensusState {
   ];
 }
 
+// Evidence is a wrapper over tendermint's DuplicateVoteEvidence
+// that implements Evidence interface expected by ICS-02
+message Evidence {
+  option (gogoproto.goproto_getters)  = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  string client_id = 1 [(gogoproto.moretags) = "yaml:\"client_id\""];
+  string chain_id  = 2 [(gogoproto.moretags) = "yaml:\"chain_id\""];
+  Header header_1  = 3 [
+    (gogoproto.customname) = "Header1",
```

Comment:
cc/ @amaurymartiny . I'm using customname here because otherwise the generated files will output "Header_1". We don't need gRPC calls for evidence because we don't store them in state (we don't even have gRPC calls for this sub-module). Just wanted to check this won't cause issues in relation to #7033 changes. I kept the "Id" naming for other fields in this struct for consistency

Reply:
If this is not ok I can update to "HeaderOne"

Reply:
I think it's fine 👍